packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -370,3 +379,18 @@ export type BufferOptions = {
    */
   readonly prioritizeTimeOverSizeThreshold?: boolean;
 };
+
+/**
+ * Specifies the audio mode that the player should use. Audio mode is set on per-app basis, if there are multiple players playing and
+ * have different a `AudioMode` specified, the highest priority mode will be used. Priority order: 'doNotMix' > 'auto' > 'duckOthers' > 'mixWithOthers'.
+ *
+ * - `mixWithOthers`: The player will mix its audio output with other apps.
+ * - `duckOthers`: The player will lower the volume of other apps if any of the active players is outputting audio.
+ * - `auto`: The player will allow other apps to keep playing audio only when it is muted. On iOS it will always interrupt other apps when `showNowPlayingNotification` is `true` due to system requirements.
+ * - `doNotMix`: The player will pause playback in other apps, even when it's muted.
+ *
+ * > On iOS, the Now Playing notification is dependent on the audio mode. If the audio mode is different from `doNotMix` or `auto` this feature will not work.
+ *
+ * @default 'auto'
```

Comment:
Nit, default value should be specified only in callside comment, since type do not have the default value by itself.